posthog/api/feature_flag.py

```diff
@@ -113,6 +115,12 @@ class FeatureFlagViewSet(StructuredViewSetMixin, AnalyticsDestroyModelMixin, vie
     queryset = FeatureFlag.objects.all()
     serializer_class = FeatureFlagSerializer
     permission_classes = [IsAuthenticated, ProjectMembershipNecessaryPermissions]
+    authentication_classes = [
+        PersonalAPIKeyAuthentication,
+        TemporaryTokenAuthentication,
+        authentication.SessionAuthentication,
+        authentication.BasicAuthentication,
+    ]
```

Comment:
What's the effective change?

Reply:
This adds `TemporaryTokenAuthentication`, and that allows the toolbar to authenticate via the tokens passed in the URL. I added a comment to make it clearer.